evalml/automl/automl_search.py

```diff
@@ -438,6 +438,7 @@ def search(self, X, y, data_checks="auto", feature_types=None, show_iteration_pl
             max_iterations=self.max_iterations,
             allowed_pipelines=self.allowed_pipelines,
             tuner_class=self.tuner_class,
+            text_columns=text_columns,
```

Comment:
Let's use [woodwork](https://woodwork.alteryx.com/en/stable/start.html) for this!

```
text_column_vals = X.select('natural_language')
text_columns = text_columns.to_pandas().columns
```